src/server/pfs/server/driver.go

```diff
@@ -60,6 +62,8 @@ const (
 
 	// maxInt is the maximum value for 'int' (system-dependent). Not in 'math'!
 	maxInt = int(^uint(0) >> 1)
+	// tmpPrefix is for temporary object paths that store merged shards.
+	tmpPrefix = "tmp"
```

Comment:
When / how are temporary objects cleaned up? Are they under the same GC system as the rest of the chunks?

Reply:
Yeah, they should be cleaned up by the gc system. There should be a set of delete calls to the gc system after a compaction is complete (which created these intermediate file sets).